tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/IContainsMessageAttributes.cs

```diff
@@ -15,6 +16,15 @@ internal interface IContainsMessageAttributes
         /// <summary>
         /// Gets or sets the message attributes
         /// </summary>
-        IDictionary MessageAttributes { get; set;  }
+        IDictionary MessageAttributes { get; set; } // <string, IMessageAttributeValue>
+    }
+
+    internal interface IMessageAttributeValue
+    {
+        string DataType { get; set; } // can be String, Number, or Binary
```

Comment:
I'd suggest adding `#nullable enable` to the top and annotating all these based on what they could properties could technically be (which is almost always nullable _unless_ they're a struct) to be on the safe side